forage-android/src/main/java/com/joinforage/forage/android/ui/ForagePANEditText.kt

```diff
@@ -93,13 +93,13 @@ class ForagePANEditText @JvmOverloads constructor(
                                 LayoutParams(LayoutParams.MATCH_PARENT, LayoutParams.WRAP_CONTENT)
 
                             // Set stroke color on focused text
-                            boxStrokeColor = focusedBoxStrokeColor
+                            boxStrokeColor = panBoxStrokeColor
                             // Set stroke width on focused text
-                            boxStrokeWidthFocused = focusedBoxStrokeWidth.toInt()
+                            boxStrokeWidthFocused = panFocusedBoxStrokeWidth.toInt()
                             // Set stroke width on unfocused text
-                            boxStrokeWidth = strokeWidth.toInt()
+                            boxStrokeWidth = panBoxStrokeWidth.toInt()
                             // Match hint text color with stroke color
-                            hintTextColor = ColorStateList.valueOf(focusedBoxStrokeColor)
+                            hintTextColor = ColorStateList.valueOf(panBoxStrokeColor)
```

Comment:
this used to for the focused state. Is this right?

Reply:
It was added by Milos, but I just removed it and tested and saw no visible differences soooooo going to remove it. Can always add it back later if GoPuff wants it.